mne/proj.py

```diff
@@ -384,9 +385,15 @@ def sensitivity_map(fwd, projs=None, ch_type='grad', mode='fixed', exclude=[],
     if mode in ['fixed', 'free']:
         sensitivity_map /= np.max(sensitivity_map)
 
-    vertices = [fwd['src'][0]['vertno'], fwd['src'][1]['vertno']]
     subject = _subject_from_forward(fwd)
-    stc = SourceEstimate(sensitivity_map[:, np.newaxis],
-                         vertices=vertices, tmin=0, tstep=1,
-                         subject=subject)
+    if len(fwd['src']) == 1:  # volume source space
+        vertices = fwd['src'][0]['vertno']
+        stc = VolSourceEstimate(sensitivity_map[:, np.newaxis],
+                                vertices=vertices, tmin=0, tstep=1,
+                                subject=subject)
+    else:
+        vertices = [fwd['src'][0]['vertno'], fwd['src'][1]['vertno']]
+        stc = SourceEstimate(sensitivity_map[:, np.newaxis],
+                             vertices=vertices, tmin=0, tstep=1,
+                             subject=subject)
```

Comment:
and then of course you can DRY this out a bit by setting only vars you need to above (`vertices` and `SEClass` [or so]) and then using the variables in one place to make the new object